apps/src/applab/applab.js

```diff
@@ -295,7 +295,8 @@ Applab.getHtml = function() {
 };
 
 Applab.getLibraries = function() {
-  return getStore().getState().applabLibrary.libraries;
+  var libraries = getStore().getState().applabLibrary.libraries;
+  return libraries.length ? libraries : undefined;
```

Comment:
is there a specific reason for this to return undefined rather than an empty array?